datasets/historical/eia_861_annual_energy_use_state_sector/dimensions/scenario.csv

```diff
@@ -0,0 +1,2 @@
+id,name
+refrence,Reference
```

Comment:
The id is misspelled. Also, based on the name of the dimension in the dataset.json5, I was expecting decarb_high, not reference. Either way will work, the files just need to be consistent.